### PR TITLE
Align Makefile with other flotta projects

### PR DESCRIPTION
**Pull request by arielireni**
Signed-off-by: arielireni <aireni@redhat.com>

### CODE REVIEW

**Review by bardielle — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -64,6 +64,12 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Docker command to use, can be podman
+DOCKER ?= docker
```
[the remark this answers is not in the document]
Comment (arielireni) [reply Makefile:68]: The purpose of this PR is to align the Makefile with other flotta projects and that's why I added these two lines in the first place (like [here](https://github.com/project-flotta/flotta-operator/blob/main/Makefile#L21) for example). I understand your point, but we aren't breaking the consistency we would like to achieve with this implementation?

[the remark this answers is not in the document]
Comment (ygalblum) [reply Makefile:68]: You're right about the consistency. But, what's the point if we're defaulting to the runner (docker) that we don't use. So, while your change allows us to replace `docker` with `podman` it still forces us to explicitly change it.

File: Makefile [hunk 2]
```diff
@@ -64,6 +64,12 @@ endif
...
+
+# Kubectl command
+KUBECTL ?= kubectl
```
[the remark this answers is not in the document]
Comment (arielireni) [reply Makefile:71]: And which of them should we give priority to in this case?

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (arielireni) [reply Makefile]: Okay, done

**Pull request merged**